gatorgrade/output/output_functions.py

```diff
@@ -1,7 +1,57 @@
-"""Add docstring."""
+"""
+This class is used for storing the main functions requested from the Github.
 
+Issue Tracker for the output team.
+For instance, functions dealing with percentage output, description output,
+and colorization of text.
+"""
+import colorama as color
 
-# This class is used for storing the main functions requested
-# from the Github Issue Tracker for the output team.
-# For instance, functions dealing with percentage output,
-# description output, and colorization of text.
+color.init()
+
+
+def sort_checks_by_result(results):
+    """
+    Process results and determine if the check passed or failed.
+
+    Argument `results` is: list[(check_result_element1, check_result_element2),(...)]]
+    """
+    passed_checks = []
+    failed_checks = []
+    # iterate through results tuples
+    for i in results:

```

Comment:
I would rename the `i` and `j` variables in the loops here to a more descriptive name for better readability.  `i` could be changed to `result`.  Also, rather than looping through `i`, you could instead do : 

```
for i in results:
    pass_check = i[1]
    if pass_check is True:
        passed_checks.append(i)
    else:
        failed_checks.append(i)
```

Since the second element in the tuple will always be a boolean.

Reply:
I added the proposed changes, thanks!